### ISSUE TITLE
[Feature request] Support mysql --login-path

### ISSUE THREAD

**Issue by sjn** [sqitchers/sqitch#590]
Hei,

I've been recently made aware of the `mysql-config-editor` tool, and it's corresponding `--login-path=name` argument to `mysql`. It seems to me that there could be some conceptual (and convenient!) overlap between this and sqitch targets. 

Is this something that could/should be implemented in sqitch?

This feature is available in mysql-5.6 and later.

Docs: https://dev.mysql.com/doc/refman/8.0/en/mysql-config-editor.html

**Comment by theory** [comment]
Yeah, `mysql` already will, but not Sqitch's use of DBD::mysql. Happy to see this added if anyone has the TUITs.
